### ISSUE TITLE
508-defect-1 ❗ Launchblocker: Link to individual debt letters fails to specify their target

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#27556]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

Issue Title: 508 defect 1, Launchblocker! Link to individual debt letters fails to specify target
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect links to downloadable documents to specify which documents they link to..

<h2>Details</h2>
Staging user 1 has two debts. When I click on Download Debt Letters, I am presented with four download links, two of which are labeled "Download First Demand Letter" and two of which are labeled "Download Second Demand Letter." The links to not specify which of the two debts they are referring to, causing confusion for screen reader users.

<h2>Acceptance Criteria</h2>
All links specify target as well as purpose.
